source/templates/code_signing.slim

```diff
@@ -89,7 +89,7 @@
 						.menu
 							button.menu-option[ng-repeat="menu in provProfileCtrl.menus" ng-click="provProfileCtrl.selectedMenu = menu" class="{{ menu.cssClass }}" ng-class="{'selected': menu == provProfileCtrl.selectedMenu}"]
 								span.title[ng-bind="menu.title"]
-								span.status[ng-class="menu.id == 'matching-certificates' ? provProfileCtrl.identitiesStatusCSSclass() : (menu.id == 'devices' ? provProfileCtrl.devicesStatusCSSclass() : '')"] = svg("dot")
+								span.status[ng-class="provProfileCtrl.identitiesStatusCSSclass()"] = svg("dot")
```

Comment:
No. This will show the identities status (ok/in-progress/missing) for every menubutton, even the devices menubutton.

Reply:
No

Reply:
check the scss please

Reply:
true